docs/user/pull-requests.rst

```diff
@@ -40,43 +52,60 @@ Privacy levels
 
    Privacy levels are only supported on :doc:`/commercial/index`.
 
-By default all docs built from pull requests are private.
+By default, all docs built from pull requests are private.
 To change their privacy level:
 
 #. Go to your project dashboard
-#. Go to :guilabel:`Admin > Advanced settings`
+#. Go to :guilabel:`Admin`, then :guilabel:`Advanced settings`
```

Comment:
I think we agreed on using the `>` syntax and we have been using it a lot on multiple pages.

Reply:
I might have missed this conversation, but would say that `>` is acceptable, but verbose language is preferable -- neither are wrong though. Microsoft has the clearest style guide here:
https://docs.microsoft.com/en-us/style-guide/procedures-instructions/describing-interactions-with-ui

Using `>` is acceptable, but preference is on using verbose language. The issues with using `>` are accessibility concerns. The `>` may not render correctly for RTL site translations, and likely sounds awful to screen readers.

Here's the page in Hebrew for example, note Google does invert the direction of `>` in some cases:

![image](https://user-images.githubusercontent.com/1140183/167694083-ce0ee947-1e80-4569-8720-609e1601cafb.png)

So, either way, this should not be ```:guilabel:`Admin > Advanced settings` ``` as these are two separate GUI elements. But also, RTL site translation breaks in this case.

Reply:
Good analysis! 💯 

> So, either way, this should not be `` :guilabel:`Admin > Advanced settings` `` as these are two separate GUI elements.

Definitely, not this option 😄